pants

```diff
@@ -57,7 +57,7 @@ source "${HERE}/build-support/pants_venv"
 # shellcheck source=build-support/bin/native/bootstrap_code.sh
 source "${HERE}/build-support/bin/native/bootstrap_code.sh"
 
-export PY="${PY:-python3}"
+export PY="${PY:-python3.7}"
```

Comment:
I would think these new PY defaults would break alot of devs un-necessarily. I happen to have 3.7, but only because I made an effort to have ~every version of python installed >=2.7 on my machine. Since this is for devs what do you think about sticking with python3 and letting the failure just happen lazily on the presumption that a dev can handle working things out? The alternative of lifting some of the pantsbuild/setup machinery into build-support/common.py is probably the way to go but may be more work than you want to glob into this PR.

Reply:
My assumption here is alot of devs have 3.8 and not 3.7.

Reply:
My `python3` points to `python3.6`, and I know I'll get very frustrated by that. So, sounds good to copy over some of `pantsbuild/setup`.

Reply:
Yeah - I had to alter my path to exclude Python 3.9 when working on the 1.30.x branch since that branch uses an old enough version of distributions that python_requires did not include 3.9 for some of them.